dydx3/modules/private.py

```diff
@@ -141,6 +141,7 @@ def update_user(
         is_sharing_username=None,
         is_sharing_address=None,
         country=None,
+        language_code=None,
```

Comment:
Will adding this cause any breaking changes in BE that we haven't tested yet?

Reply:
@ruixhuang has already tested this endpoint in staging, so there are no breaking changes - it's an optional field.